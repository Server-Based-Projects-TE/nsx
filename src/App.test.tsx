// import React from 'react'
// import ReactDOM from 'react-dom/client'
//
// import App from './App'
//
// test('App is render without crash', () => {
//   const container = window.document.createElement('div')
//   const root = ReactDOM.createRoot(container)
//   root.render(<App />)
//   expect(container).toMatchSnapshot()
// })
// @ts-ignore
test('TODO', () => {
  expect(true).toMatchSnapshot()
})
