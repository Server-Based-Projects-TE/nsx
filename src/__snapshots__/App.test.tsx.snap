// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TODO 1`] = `true`;
